modules/v1/users/controllers/AuthController.php

```diff
@@ -23,10 +23,10 @@ public function actionLogin()
         $request = Yii::$app->request;
         $model = new LoginForm();
 
-        if (!$model->load($request->post(), '') || !$model->validate() || !$model->login()) {
+        if (!$model->load($request->post(), '') || $model->isNotActiveUser($request->post('username')) || !$model->validate() || !$model->login()) {
```

Comment:
@skavt  It is better to create custom validation for password attribute and set the error there, instead of calling separate isNotActiveUser method.